LICENSES/LicenseRef-Slint-commercial.md

```diff
@@ -1,6 +1,393 @@
+    be recreated, then in SixtyFPS’s sole discretion, SixtyFPS may ask Customer to provide
+    the necessary setup or elect to recreate the problem at the Customer location. Any costs incurred
+    will be invoiced to Customer.
+
+9.  **Errors in Modified Source**
+
+    SixtyFPS is not responsible for errors introduced by Customer
+    modifications to Licensed Software. Defects must be demonstrated on
+    an unmodified version of Licensed Software provided by SixtyFPS.
+    
+10. **Non Licensed Software related problems**
+
+    SixtyFPS will make reasonable efforts to provide workarounds for problems that are not directly related to the Licensed Software but are present in the Customer's Products built with the Licensed Software. Customer understands that workarounds may not be possible for all problems.
+
+11. **Operating Systems Updates**
+
+    SixtyFPS shall consider making commercially reasonable efforts to provide Customers with updates to the Licensed Software for newly released versions of operating systems. If, however, extensive changes to the Licensed Software are required, SixtyFPS may charge additional fees for the changes. Customer is under no obligation to accept such updates that require additional fees and SixtyFPS is not obligated to provide any specific operating system updates.
+
+# Exhibit C - Customer's Products
+
+The list of Customer's Products that are designed, developed and tested using the
+Licensed Software and are allowed to be distributed in conjunction with
+the object code of the Licensed Software.
+
+-   \<enter list of products>
+
+# Exhibit D - Use of the Licensed Software
+
+1. **On payment of applicable Fees**
+
+The license grants are conditional on the full and timely payment of applicable Fees; if said Fees are not 
+paid in full and at their due date, the licenses shall not be granted and the Customer's Products shall be 
+deemed as made in breach of this Agreement ex tunc.
+
+2.  **Under the Ambassador Program**
+
+The license grants are conditional on the fullfilment of the obligations set forth below; if said obligations are not fulfilled, the licenses shall not be granted and the Customer's Products shall be deemed as made in breach of this Agreement ex tunc.
+
+Obligations:
+   1. Customer grants SixtyFPS the right to use Customer's name and logo in marketing material,
+   2. Customer will include reasonably visible notices in the documentation, marketing materials and social 
+      media channels related to Customer's Products that such were made with the Licensed Software,
+   3. Customer agrees to provide a written testimonial statement about the use of the Licensed Software.
+
+
+**Customer has been granted the right to use the Licensed Software [on payment of applicable Fees / under the Ambassador Program].** (cross out the incorrect option) 
+The permitted number of User Seats and the distribution of Customer's Products is specified below
```

Comment:
Should this be replaced with some generic blob about seat/product based licenses?

This seems specific to one customer, and does into a generic statement IMHO,

Reply:
The original agreement was in a format that required signatures from both parties. As @ogoffart also mentioned, maybe we should edit the agreement so that it does not require signatures anymore. One way that I can think of is using a license certificate that is issued to each customer and which mentions the scope of the license grant (whether Ambassador or User Seats (Flex) or Buyout)